extensions/guacamole-auth-quickconnect/src/main/resources/templates/quickconnectField.html

```diff
@@ -0,0 +1,9 @@
+<meta name="before" content=".recent-connections" />
+<div class="header" ng-controller="quickconnectController">
+    <div class="quickconnect-container">
+        <form ng-submit="quickConnect()">
+            <input type=text class="quickconnect-field" placeholder="Enter Connection URI" ng-model="uri" />
```

Comment:
Strings which will be displayed to the user need to be defined as translation strings, rather than hard-coded in the HTML.

Reply:
Done.